src/popup/router/pages/Account.vue

```diff
@@ -37,6 +37,9 @@
         <BoxButton text="Activity" to="/transactions" class="tour__step5">
           <Activity slot="icon" />
         </BoxButton>
+        <BoxButton :text="$t('pages.appVUE.invite')" to="/invite">
```

Comment:
I'm proposing to always refer the route by name:
```suggestion
        <BoxButton :text="$t('pages.appVUE.invite')" :to="{ name: 'invite' }">
```